build_test.go

```diff
@@ -1343,20 +1346,20 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 		})
 
 		when("Publish option", func() {
-			when("true", func() {
-				var remoteRunImage *fakes.Image
+			var remoteRunImage *fakes.Image
 
-				it.Before(func() {
-					remoteRunImage = fakes.NewImage("default/run", "", nil)
-					h.AssertNil(t, remoteRunImage.SetLabel("io.buildpacks.stack.id", defaultBuilderStackID))
-					h.AssertNil(t, remoteRunImage.SetLabel("io.buildpacks.stack.mixins", `["mixinA", "mixinX", "run:mixinZ"]`))
-					fakeImageFetcher.RemoteImages[remoteRunImage.Name()] = remoteRunImage
-				})
+			it.Before(func() {
+				remoteRunImage = fakes.NewImage("default/run", "", nil)
+				h.AssertNil(t, remoteRunImage.SetLabel("io.buildpacks.stack.id", defaultBuilderStackID))
+				h.AssertNil(t, remoteRunImage.SetLabel("io.buildpacks.stack.mixins", `["mixinA", "mixinX", "run:mixinZ"]`))
+				fakeImageFetcher.RemoteImages[remoteRunImage.Name()] = remoteRunImage
+			})
 
-				it.After(func() {
-					remoteRunImage.Cleanup()
-				})
+			it.After(func() {
+				remoteRunImage.Cleanup()
+			})
 
+			when("true", func() {
```

Comment:
The groupings were messed up in this file - all tests below were nested under this group.